docs/app/pages/components/sections/tree-view/tree-view-ng1/snippets/sample.html

```diff
@@ -1 +1,15 @@
-<tree-view data="vm.data" selected="vm.selected" add-item="vm.addItem" icons="vm.iconConfig" delete="vm.deleteFn" options="vm.treeOptions"></tree-view>
\ No newline at end of file
+<tree-view data="vm.data" selected="vm.selected" add-item="vm.addItem" icons="vm.iconConfig" delete="vm.deleteFn" options="vm.treeOptions"></tree-view>
+
+<div class="row uxd-customize-example">
```

Comment:
I think we could probably remove this as a custom class won't be a feature that is used very often. Just might allow us to keep the sample code more simple and concise making it easier for teams to understand.